core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7468UnsupportedPluginsParametersTest.java

```diff
@@ -0,0 +1,183 @@
+    {
+        List<String> warnLines = performTest( "config-build-mixed" );
+        assertWarningContains( warnLines );
+    }
+
+    /**
+     * Test that ensures that warning is printed for configuration
+     */
+    public void testInPluginManagement() throws Exception
+    {
+        List<String> warnLines = performTest( "config-plugin-management" );
+        assertWarningContains( warnLines );
+    }
+
+    /**
+     * Test that ensures that warning is printed for configuration
+     */
+    public void testInPluginManagementParent() throws Exception
+    {
+        List<String> warnLines = performTest( "config-plugin-management-parent" );
+        assertWarningContains( warnLines );
+    }
+
+    /**
+     * Test that ensures that warning is printed for configuration
+     */
+    public void testWithForkedGoalExecution() throws Exception
+    {
+        List<String> warnLines = performTest( "config-with-fork-goal" );
+
+        assertTrue( warnLines.remove(
+            "[WARNING] Parameter 'invalidXml' is unknown for plugin 'maven-it-plugin-fork:2.1-SNAPSHOT:fork-goal (fork)'" ) );
+
+        assertTrue( warnLines.remove(
+            "[WARNING] Parameter 'invalidValue' is unknown for plugin 'maven-it-plugin-fork:2.1-SNAPSHOT:fork-goal (fork)'" ) );
+
+        assertTrue( warnLines.remove(
+            "[WARNING] Parameter 'invalidXml' is unknown for plugin 'maven-it-plugin-fork:2.1-SNAPSHOT:touch (touch)'" ) );
+
+        assertTrue( warnLines.remove(
+            "[WARNING] Parameter 'invalidValue' is unknown for plugin 'maven-it-plugin-fork:2.1-SNAPSHOT:touch (touch)'" ) );
+
+        assertTrue( "Not verified line: " + warnLines, warnLines.isEmpty() );
+    }
+
+    private List<String> performTest( String project ) throws Exception
+    {
+        File testDir = ResourceExtractor.simpleExtractResources( getClass(), "/mng-7468-unsupported-params" );
+
+        Verifier verifier = newVerifier( new File( testDir, project ).getAbsolutePath() );
+        verifier.executeGoal( "validate" );
+        verifier.verifyErrorFreeLog();
+        verifier.resetStreams();
+
+        List<String> logLines = verifier.loadFile( verifier.getBasedir(), verifier.getLogFileName(), false );
+        return findUnknownWarning( logLines );
+    }
+
+    private void assertWarningContains( List<String> warnLines )
+    {
+        assertTrue( warnLines.remove(
+            "[WARNING] Parameter 'invalidValue' is unknown for plugin 'maven-it-plugin-configuration:2.1-SNAPSHOT:touch (default)'" ) );
+
+        assertTrue( warnLines.remove(
+            "[WARNING] Parameter 'invalidXml' is unknown for plugin 'maven-it-plugin-configuration:2.1-SNAPSHOT:touch (default)'" ) );
+
+        assertTrue( "Not verified line: " + warnLines, warnLines.isEmpty() );
+    }
+
+    private List<String> findUnknownWarning( List<String> logLines )
+    {
+        Pattern pattern = Pattern.compile( "\\[WARNING] Parameter .* is unknown.*" );
```

Comment:
WE rather require content: `.+`, no?

Reply:
in this case will be the same ... we searching lines of this pattern and in next step specific line values are compared

Reply:
OK, I see.